test/functionality/r-bridge/lang/ast/parse-access.ts

```diff
@@ -342,157 +342,409 @@ describe('Parse value access', withShell(shell => {
+					arguments: [{
+						type:      RType.Symbol,
+						location:  rangeFrom(1, 1, 1, 1),
+						namespace: undefined,
+						lexeme:    'b',
+						content:   'b',
+						info:      {}
+					}, {
+						type:     RType.Argument,
+						location: rangeFrom(1, 4, 1, 4),
+						lexeme:   '5',
+						name:     undefined,
+						info:     {},
+						value:    {
+							type:     RType.Number,
+							location: rangeFrom(1, 4, 1, 4),
+							lexeme:   '5',
+							content:  numVal(5),
+							info:     {}
+						}
+					}, {
+						type:     RType.Argument,
+						location: rangeFrom(1, 6, 1, 6),
+						lexeme:   '3',
+						name:     undefined,
+						info:     {},
+						value:    {
+							type:     RType.Number,
+							location: rangeFrom(1, 6, 1, 6),
+							lexeme:   '3',
+							content:  numVal(3),
+							info:     {}
+						}
+					}
+					]
+				})
+			}
+		])
+		assertAst('Multiple with empty', shell, 'b[[5,,]]', [
+			{
+				step:   NORMALIZE,
+				wanted: exprList({
+					type:     RType.Access,
+					location: rangeFrom(1, 2, 1, 3),
+					lexeme:   '[[',
+					operator: '[[',
+					info:     {},
+					accessed: {
+						type:      RType.Symbol,
+						location:  rangeFrom(1, 1, 1, 1),
+						namespace: undefined,
+						lexeme:    'b',
+						content:   'b',
+						info:      {}
+					},
+					access: [{
 
-				type:     RType.Argument,
-				location: rangeFrom(1, 4, 1, 4),
-				lexeme:   '5',
-				name:     undefined,
-				info:     {},
-				value:    {
-					type:     RType.Number,
-					location: rangeFrom(1, 4, 1, 4),
-					lexeme:   '5',
-					content:  numVal(5),
-					info:     {}
-				}
-			},null,null]
-		}))
+						type:     RType.Argument,
+						location: rangeFrom(1, 4, 1, 4),
+						lexeme:   '5',
+						name:     undefined,
+						info:     {},
+						value:    {
+							type:     RType.Number,
+							location: rangeFrom(1, 4, 1, 4),
+							lexeme:   '5',
+							content:  numVal(5),
+							info:     {}
+						}
+					}, null, null]
+				})
+			},
+			{
+				step:   DESUGAR_NORMALIZE,
+				wanted: exprList({
+					type:         RType.FunctionCall,
+					location:     rangeFrom(1, 2, 1, 3),
+					lexeme:       '[[',
+					info:         {},
+					functionName: {
+						type:      RType.Symbol,
+						lexeme:    '[[',
+						content:   '[[',
+						info:      {},
+						location:  rangeFrom(1, 2, 1, 3),
+						namespace: undefined
+					},
+					flavor:    'named',
+					arguments: [{
+						type:      RType.Symbol,
+						location:  rangeFrom(1, 1, 1, 1),
+						namespace: undefined,
+						lexeme:    'b',
+						content:   'b',
+						info:      {}
+					}, {
+						type:     RType.Argument,
+						location: rangeFrom(1, 4, 1, 4),
+						lexeme:   '5',
+						name:     undefined,
+						info:     {},
+						value:    {
+							type:     RType.Number,
+							location: rangeFrom(1, 4, 1, 4),
+							lexeme:   '5',
+							content:  numVal(5),
+							info:     {}
+						}
+					}, undefined, undefined]
```

Comment:
Let's make an issue for this, we should not use `undefined` for missing arguments but a special value to avoid forgetting them :) maybe something like `empty`.

Reply:
#599

Reply:
Great!